### DISCUSSION TITLE
No UI pages for RLS policies for non-public schemas visible?

### DISCUSSION THREAD

**Discussion by dominikjannis** [supabase/supabase#11862] [Questions] [answered]
When configuring RLS policies in Supabase Studio, only tables in the public schema are shown. Do i miss something here, or is it not possible yet to create policies via the UI for other database schemas?

**Comment by GaryAustin1** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/54564956/214090744-b4bac88c-bcc4-4936-8fbe-901d5d1aac2a.png)

**Reply by dominikjannis** [reply]
i looked in the wrong place, thanks :+1:
